src/actions/register.js

```diff
@@ -0,0 +1,56 @@
+var { API_URL } = require('../PLEnv');
+var { Action, ThunkAction } = require('./types');
+
+function findByUsername(username: string){
+    return new Promise((resolve, reject) => {
+        fetch(`https://api-staging.powerli.ne/api-public/users/?username=` + username, {
```

Comment:
I do not think the "https://api-staging.powerli.ne" should be here> I think there's supposed to be a flexible reference that we can change at a master level when we are deciding which environment to create a build for... @awis0330 can confirm